Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py

```diff
@@ -382,74 +189,102 @@ def get_company_details_command(client, args):
-                                                    subIndustry=res_json.get('sub_industry'),
-                                                    subIndustrySlug=res_json.get('sub_industry_slug'),
-                                                    homePage=res_json.get('homepage'),
-                                                    primaryDomain=res_json.get('primary_domain'),
-                                                    type=res_json.get('type'),
-                                                    displayURL=res_json.get('display_url'),
-                                                    ratingDetails=res_json.get('rating_details'),
-                                                    ratings=res_json.get('ratings'),
-                                                    searchCount=res_json.get('search_count'),
-                                                    subscriptionType=res_json.get('subscription_type'),
-                                                    sparkline=res_json.get('sparkline'),
-                                                    subscriptionTypeKey=res_json.get('subscription_type_key'),
-                                                    subscriptionEndDate=res_json.get('subscription_end_date'),
-                                                    bulkEmailSenderStatus=res_json.get('bulk_email_sender_status'),
-                                                    serviceProvider=res_json.get('service_provider'),
-                                                    customerMonitoringCount=res_json.get('customer_monitoring_count'),
-                                                    availableUpgradeTypes=res_json.get('available_upgrade_types'),
-                                                    hasCompanyTree=res_json.get('has_company_tree'),
-                                                    hasPreferredContact=res_json.get('has_preferred_contact'),
-                                                    isBundle=res_json.get('is_bundle'),
-                                                    ratingIndustryMedian=res_json.get('rating_industry_median'),
-                                                    primaryCompany=res_json.get('primary_company'),
-                                                    permissions=res_json.get('permissions'),
-                                                    isPrimary=res_json.get('is_primary'),
-                                                    securityGrade=res_json.get('security_grade'),
-                                                    inSpmPortfolio=res_json.get('in_spm_portfolio'),
-                                                    isMycompMysubsBundle=res_json.get('is_mycomp_mysubs_bundle'),
-                                                    companyFeatures=res_json.get('company_features')
-                                                    )
+        generic_iam_context = {
+            'guid': res_json.get('guid'),
```

Comment:
I still don't understand why you need to re-create this dictionary - isn't this exactly how the res_json looks like? why not just passing it as output?
The json original keys are perfectly fine as outputs

Reply:
For output context path in command I have used this naming convention, hence I'm changing names in code also.